src/Symfony/Bundle/FrameworkBundle/Resources/config/routing.xml

```diff
@@ -112,13 +112,13 @@
             <argument>%kernel.debug%</argument>
         </service>
 
-        <service id="Symfony\Bundle\FrameworkBundle\Controller\RedirectController" public="true">
+        <service id="framework.controller.redirect" class="Symfony\Bundle\FrameworkBundle\Controller\RedirectController" public="true">
```

Comment:
I think we need to keep this and the next one, not for autowiring, but to ease routing